velox/connectors/hive/tests/HiveDataSinkTest.cpp

```diff
@@ -192,6 +205,58 @@ class HiveDataSinkTest : public exec::test::HiveConnectorTestBase {
     connectorQueryCtx_ = std::move(connectorQueryCtx);
   }
 
+  template <typename ReaderType, dwio::common::FileFormat format>
+  void testFlushPolicy() {
```

Comment:
I don't see a lot of reuse considering the idefs. Why not add separate tests for DWRF and Parquet in the corresponding WriterTest?

Reply:
@majetideepak Yes. Seperate the tests for DWRF and Parquet.